app/src/main/java/org/kiwix/kiwixmobile/main/KiwixNewNavigationActivity.kt

```diff
@@ -82,7 +82,7 @@ class KiwixNewNavigationActivity : CoreMainActivity() {
   override fun onBackPressed() {
     supportFragmentManager.fragments.filterIsInstance<BaseFragmentActivityExtensions>().forEach {
       if (it.onBackPressed(this) == BaseFragmentActivityExtensions.Super.ShouldCall) {
-        super.onBackPressed()
+        finish()
```

Comment:
I want to check if the current fragment is a top level destination and otherwise call super but the only reliable way I found is to compare the current destination ID with all top level destinations which seems silly. https://stackoverflow.com/questions/52111516/in-android-navigation-architecture-how-can-i-check-if-current-fragment-is-the-l